### PR TITLE
packages: drop support for Ubuntu 18.04

### PR DESCRIPTION
**Pull request by komainu8**
Because Ubuntu 18.04 reached EOL.

### CODE REVIEW

File: packages/mysql-5.7-mroonga/Rakefile
Comment (kou) [inline packages/mysql-5.7-mroonga/Rakefile:1]: Can we remove this directory?

Comment (komainu8) [reply packages/mysql-5.7-mroonga/Rakefile:1]: Thanks.
I've fixed it.

Comment (kou) [reply packages/mysql-5.7-mroonga/Rakefile:1]: Could you also update https://github.com/mroonga/mroonga/blob/master/packages/Rakefile#L11 ?

Comment (komainu8) [reply packages/mysql-5.7-mroonga/Rakefile:1]: Ah...Sorry.
I've fixed it

Comment (kou) [reply packages/mysql-5.7-mroonga/Rakefile:1]: We may want to run `cd packages && rake version:update` in a CI job to detect this case automatically.

**Review by kou — APPROVED** [review]
+1

**Comment by komainu8** [comment]
Thank you for your comments!

**Pull request merged**